woodpecker-go/woodpecker/client.go

```diff
@@ -215,6 +215,18 @@ func (c *client) BuildList(owner, name string) ([]*Build, error) {
 	return out, err
 }
 
+func (c *client) BuildCreate(owner, name, branch string, variables map[string]string) (*Build, error) {
+	type ManualBuildReq struct {
```

Comment:
Type should be declared in types.go

Reply:
I also would not call it manual build option, as if we gona add otjer events support it is just "create a build", so this func should get a option struct